geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/Runner.scala

```diff
@@ -153,4 +156,22 @@ trait Runner extends LazyLogging {
   }
 }
 
-case class AutocompleteInfo(path: String, commandName: String)
+object Runner {
+
+  private var added = false
+
+  /**
+   * Add jcommander converter factories. Note that the implementation is a static,
+   * shared, non-thread-safe list, so we have to synchronize ourselves.
```

Comment:
Ok, how did you run into this?

Reply:
CI kept failing on the export test... dug into it and found this.

Reply:
in normal use i don't think it would ever happen since we only load 1 runner at a time.